src/main/java/com/snowflake/kafka/connector/SnowflakeSinkTask.java

```diff
@@ -450,6 +450,10 @@ private KafkaRecordErrorReporter createKafkaRecordErrorReporter() {
               (record, error) -> {
                 try {
                   // Blocking this until record is delivered to DLQ
+                  DYNAMIC_LOGGER.debug(
```

Comment:
Looks like we don't have any tests that go into this branch?

Reply:
We have an in memory queue implementation which is tested. 

It's hard to test this in Java. But let me see if we can add more coverage for this.

Reply:
Yes, this is essentially a no-op. E2E tests covers this.

Reply:
Added coverage but only through E2E test.